src/frontend/src/lib/components/core/SignOut.svelte

```diff
@@ -4,10 +4,12 @@
 	import { LOGOUT_BUTTON } from '$lib/constants/test-ids.constants';
 	import { signOut } from '$lib/services/auth.services';
 	import { i18n } from '$lib/stores/i18n.store';
+	import { replaceHistory } from '$lib/utils/route.utils';
 
 	const dispatch = createEventDispatcher();
 
 	const logout = async () => {
+		replaceHistory(new URL(window.location.origin));
```

Comment:
1. Should we actually replace the history?
2. Have you tried using `goto`?
3. Can you rename the PR, it's not a `style` change but, rather a `feature`.
4. The implementation should probably belong in the `auth.service` module rather than the component, as there might be other consumers that could use the function without relying on the component. WDYT?
5. Can you provide a test, either an E2E or a unit test?

Reply:
@peterpeterparker 
1/2. You are right i guess, we should not replace the history. The `goto` might be the better solution.
3. done
4. I don't see exactly why we should move this into the `auth.service` module. It is only a simple `goto` command that is only used if the user clicks on the `logout` button.
5. Yes, i will provide a test

Reply:
I believe this is the right approach, aligning with the method we've been applying over the past few weeks/months:

1. We should provide new tests that cover the service (in that case `auth.module`) to resolve the tech debt and facilitate the addition of the new feature.
2. Then, we should extend the `logout` feature with the "result url option to navigate to the root" and adapt the tests accordingly.

Reply:
@peterpeterparker @AntonioVentilii I added the optional `resetUrl` parameter to the `SignOut` function and implemented some test cases.